common/amundsen_common/utils/atlas_utils.py

```diff
@@ -0,0 +1,245 @@
+# Copyright Contributors to the Amundsen project.
```

Comment:
@mgorsk1 you don't need to name this `atlas_utils`, as this is clear that the file is under /utils directory. 
I'd rename this to `atlas.py` only.

Reply:
Ok, i'll let @dechoma fix this :D